lib/connection.js

```diff
@@ -1452,7 +1452,7 @@ Connection.prototype.setClient = function setClient(client) {
  *
  * @param {Object} options
  * @param {Boolean} options.continueOnError `false` by default. If set to `true`, mongoose will not throw an error if one model syncing failed, and will return an object where the keys are the names of the models, and the values are the results/errors for each model.
- * @returns
+ * @return {Promise|undefined} Returns `undefined` if callback is specified, returns a promise if no callback, when the Promise resolves the value is a list of the dropped indexes.
```

Comment:
it does not contain a callback?! Also it is defined as async function, so it is always a Promise?!

Reply:
i dont really know about that, i just copied it from `lib/model.js`, where i also had slightly extended the description, and it just seemed to return everything from what `lib/model.js` returned

Reply:
updated in 407fe6194e4f81aba170b6b88dd4b987c8bdb789, is this more correct?